airbyte-cdk/python/airbyte_cdk/test/mock_http/mocker.py

```diff
@@ -60,7 +60,9 @@ def _mock_request_method(
         getattr(self._mocker, method)(
             requests_mock.ANY,
             additional_matcher=self._matches_wrapper(matcher),
-            response_list=[{"text": response.body, "status_code": response.status_code} for response in responses],
+            response_list=[
```

Comment:
Curious: what formatter are you using, and do we have centralized enforced formatting / linting yet?

Reply:
I blindly use airbyte-ci format fix all